code/datums/components/storage/storage.dm

```diff
@@ -125,7 +125,7 @@
 	if (can_hold_list != null)
 		can_hold = typecacheof(can_hold_list)
 
-	if (cant_hold_list != null)	
+	if (cant_hold_list != null)
```

Comment:
```suggestion
	if (cant_hold_list)
```

Reply:
I know it is done the wrong way just above.
No reason to make shitcode yourself